core/assets/bundles/bundle_de.properties

```diff
@@ -1155,32 +1155,32 @@ block.force-projector.description = Erzeugt ein sechseckiges Kraftfeld um sich h
 block.junction.description = Fungiert als Brücke zwischen zwei kreuzenden Förderbändern. Nützlich, wenn zwei verschiedene Förderbänder sich kreuzen, aber unterschiedliche Materialien verwenden.
 block.bridge-conveyor.description = Verbesserter Transportblock. Erlaubt es, Materialien über bis zu 3 Kacheln beliebigen Terrains oder Inhalts zu transportieren.
 block.phase-conveyor.description = Verbesserter Transportblock. Verwendet Strom, um Materialien zu einem verbundenen Phasen-Förderband über mehrere Kacheln zu teleportieren.
 block.sorter.description = Sortiert Materialien. Wenn ein Gegenstand der Auswahl entspricht, darf er vorbei. Andernfalls wird er links oder rechts ausgegeben.
-block.inverted-sorter.description = Processes items like a standard sorter, but outputs selected items to the sides instead.
+block.inverted-sorter.description = Funktioniert wie ein normaler Sortierer, aber gibt das ausgewählte Material seitwärts auss.
 block.router.description = Akzeptiert Materialien aus einer Richtung und leitet sie gleichmäßig in bis zu drei andere Richtungen weiter. Nützlich, wenn die Materialien aus einer Richtung an mehrere Empfänger verteilt werden sollen.
 block.distributor.description = Ein weiterentwickelter Verteiler, der Materialien in bis zu sieben Richtungen gleichmäßig verteilt.
 block.overflow-gate.description = Ein Verteiler, der nur Materialien nach links oder rechts ausgibt, falls der Weg gerade aus blockiert ist.
-block.underflow-gate.description = The opposite of an overflow gate. Outputs to the front if the left and right paths are blocked.
+block.underflow-gate.description = Das Gegenteil eines Überlauftors. Gibt Materialen nur nach vorne aus, wenn der Weg rechts und links blockiert ist.
 block.mass-driver.description = Ultimativer Transportblock. Sammelt mehrere Materialien und schießt sie zu einem verbundenen Massenbeschleuniger über eine große Reichweite.
 block.mechanical-pump.description = Eine günstige, langsame Pumpe, die keinen Strom benötigt.
 block.rotary-pump.description = Eine fortgeschrittene Pumpe, die mithilfe von Strom doppelt so schnell pumpt.
 block.thermal-pump.description = Die ultimative Pumpe, dreimal so schnell wie eine mechanische Pumpe und die einzige Pumpe, die Lava fördern kann.
 block.conduit.description = Standard Flüssigkeits-Transportblock. Funktioniert wie ein Förderband, nur für Flüssigkeiten. Wird am Besten mit Extraktoren, Pumpen oder anderen Kanälen benutzt.
 block.pulse-conduit.description = Verbesserter Flüssigkeits-Transportblock. Transportiert Flüssigkeiten schneller und speichert mehr als Standard Kanäle.
-block.plated-conduit.description = Moves liquids at the same rate as pulse conduits, but possesses more armor. Does not accept fluids from the sides by anything other than conduits.\nLeaks less.
+block.plated-conduit.description = Bewegt Gegenstände genauso schnell wie ein Impulskanal, ist aber besser gepanzert. Akzeptiert nur Verbindungen mit anderen Kanälen.\nHat weniger Lecks.
```

Comment:
Bewegt Flüssigkeit und keine Gegenstände.

"Leckt weniger" wäre hier treffender, würde allerdings etwas in Richtung "Verliert weniger Flüssigkeit" vorschlagen.

Reply:
Verschüttet, vergießt